### ISSUE TITLE
g10k -puppetfile deletes other stuff in moduledir

### ISSUE THREAD

**Issue by alexharv074** [xorpaul/g10k#75]
I would like to include roles and profiles in my control repo modules directory, as is common. Then I would like to use g10k to install the other modules from Puppetfile into the same directory.

It looks like r10k made this possible [here](https://tickets.puppetlabs.com/browse/RK-159). However, I see that g10k doesn't support this.

Is that something that's easy to fix?

**Comment by xorpaul** [comment]
I added support for `local` modules just like r10k supports:

```
mod 'localstuff', :local => true
[...]
```

With `-debug` you should see something like this in your output:

```
DEBUG Not deleting /tmp/example/foobar_local_modules/modules/localstuff as it is declared as a local module
```

**Comment by baurmatt** [comment]
The current README still mentions the local module as unsupported:

```
No 'local'-Modules support
```

**Comment by xorpaul** [comment]
Nice catch 👍 

Fixed with 5804e30049a3404997956c43264ebde6dc6a0fdc

**Comment by alexharv074** [comment]
Thanks for this @xorpaul
